packages/cli-plugin-metro/src/tools/getDefaultMetroConfig.ts

```diff
@@ -0,0 +1,87 @@
+import type {InputConfigT} from 'metro-config';
+import path from 'path';
+import type {ConfigLoadingContext} from './loadMetroConfig';
+
+/**
+ * This module reproduces defaults from the @react-native/metro-config package,
+ * used in ./loadMetroConfig.js to provide a soft upgrade process when upgrading
+ * to React Native 0.72.
+ *
+ * These values will be used when:
+ * - RN CLI 11.x or greater is present in a project (from React Native 0.72).
+ * - The project has not yet followed the upgrade instructions to update
+ *   metro.config.js to extend '@react-native/metro-config'.
+ *
+ * Until we remove this file in a future release, updates should be made both
+ * here and in '@react-native/metro-config'. (Note: Updates to these values are
+ * generally rare.)
+ *
+ * TODO(@huntie): Remove this file in a future React Native release.
+ */
+
+/**
+ * @deprecated (React Native 0.72.0) Defaults should be updated here and in
+ *   https://github.com/facebook/react-native/tree/main/package/metro-config/index.js
+ */
+const INTERNAL_CALLSITES_REGEX = new RegExp(
+  [
+    '/Libraries/Renderer/implementations/.+\\.js$',
+    '/Libraries/BatchedBridge/MessageQueue\\.js$',
+    '/Libraries/YellowBox/.+\\.js$',
+    '/Libraries/LogBox/.+\\.js$',
+    '/Libraries/Core/Timers/.+\\.js$',
+    '/Libraries/WebSocket/.+\\.js$',
+    '/Libraries/vendor/.+\\.js$',
+    '/node_modules/react-devtools-core/.+\\.js$',
+    '/node_modules/react-refresh/.+\\.js$',
+    '/node_modules/scheduler/.+\\.js$',
+    '/node_modules/event-target-shim/.+\\.js$',
+    '/node_modules/invariant/.+\\.js$',
+    '/node_modules/react-native/index.js$',
+    '/metro-runtime/.+\\.js$',
+    '^\\[native code\\]$',
+  ].join('|'),
+);
+
+/**
+ * Get the static Metro config defaults for a React Native project.
+ *
+ * @deprecated (React Native 0.72.0) Defaults should be updated here and in
+ *   https://github.com/facebook/react-native/tree/main/package/metro-config/index.js
+ */
+export default function getDefaultMetroConfig(
+  ctx: ConfigLoadingContext,
+): InputConfigT {
+  return {
+    resolver: {
+      resolverMainFields: ['react-native', 'browser', 'main'],
+      unstable_conditionNames: ['require', 'react-native'],
+    },
+    serializer: {
+      getPolyfills: () =>
+        require(path.join(ctx.reactNativePath, 'rn-get-polyfills'))(),
```

Comment:
Restored (https://github.com/react-native-community/cli/pull/1875/files#diff-3eb3e1e81b924a57b22a630c633912f2f8777b5a201992591c8a3517fb63f525L76-L77).

Deviates from `@react-native/metro-config`, which references the standalone package `@react-native/js-polyfills`. These are equivalent, but allows this repo to not reference the RN-hosted package (see removal in `yarn.lock`).